package.json

```diff
@@ -12,9 +12,10 @@
     "@testing-library/user-event": "^7.2.1",
     "babel-jest": "^27.0.0",
     "bootstrap": "^4.6.2",
+    "fetch-mock": "^9.11.0",
```

Comment:
unused package, please remove this line and regenerate `package-lock.json`
1. delete `package-lock.json`
2. run `npm i`
3. push changes

Reply:
when u say delete package-lock.json, do u just mean the line? or the whole file?